components/PlaySelector.vue

```diff
@@ -17,9 +18,10 @@ const props = defineProps({
 
 const playServices = usePlayServices()
 const playServicePreference = usePlayServicePreference()
-
 const selectedPlayService = ref(playServicePreference)
 
+const currentEpisode = useCurrentEpisode()
+
 const launchService = (service) => {
   //updating local storage on update
   window.localStorage.setItem("selectedPlayService", JSON.stringify(service))
```

Comment:
Would it make sense to store the string `selectedPlayService` somewhere as a constant so that whatever retrieves that item can refer to the same string?